docs/src/main/tut/optics/lens.md

```diff
@@ -101,6 +101,41 @@ val address = GenLens[Person](_.address)
 (address composeLens streetNumber).get(john)
 (address composeLens streetNumber).set(2)(john)
 ```
+### Other Ways of Lens Composition
+
+Is possible to compose few `Lenses` together by using `compose`:
```

Comment:
it might be good to mention that all `Setter` like optics offer `set` and `modify` methods that returns an `EndoFunction` (i.e. `S` => `S`) which means that we can compose modification using basic function composition.